### PR TITLE
[WIP] Simplify 'Add a theme' form

### PR DESCRIPTION
**Pull request by JevgenijVisockij**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Simplifies add a theme forms
| Type?         | refacto
| Category?     | BO
| BC breaks?    | yes
| Deprecations? | no
| Fixed ticket? | Part of #16482
| How to test?  |  Theme & Logo > Theme. Everything should work exactly the same

:notebook: **BC Break**
Backwards compatibility break introduced due to extension of TranslationAwareType by ImportThemeType. This means if any module extends any of this type they will get an exception upon upgrading to PS version containing changes in this PR.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21243)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig [hunk 1]
```diff
@@ -101,6 +101,12 @@
   {{ block('form_help') }}
 {% endblock switch_widget %}
 
+{%- block url_widget -%}
```
Comment (matks) [inline src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig]: This modification is in `prestashop_ui_kit.html.twig` and not `prestashop_ui_kit_base.html.twig` is that intended ?

Comment (JevgenijVisockij) [reply src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig]: Good catch, yeah I think it should be in base.

**Comment by JevgenijVisockij** [comment]
Blocked until https://github.com/PrestaShop/PrestaShop/pull/21878 is merged since it also deals with problem in file type.

**Comment by JevgenijVisockij** [comment]
Something went really wrong during rebase and because this PR only had 4 commits it was easier to add them again then to deal with rebase mess, so closes in favor of: 
https://github.com/PrestaShop/PrestaShop/pull/23127

**Pull request closed without merging**